src/pages/handbook.tsx

```diff
@@ -5,6 +5,32 @@ import { StaticImage } from 'gatsby-plugin-image'
 import Link from 'components/Link'
 import PostLayout from 'components/PostLayout'
 
+const chapters = [
+    {
+        name: 'Chapters',
+        links: [
+            { order: '1', name: 'What is PostHog?', to: '/handbook/what-is-posthog' },
+            { order: '2', name: 'A wide company with small teams', to: '/handbook/wide-company' },
+            {
+                order: '3',
+                name: 'Providing a world-class engineering environment',
+                to: '/handbook/world-class-engineering',
+            },
+            { order: '4', name: 'Deciding which products to build', to: '/handbook/which-products' },
+            { order: '5', name: 'How we get users', to: '/handbook/how-we-get-users' },
+            { order: '6', name: 'Enduringly low prices', to: '/handbook/low-prices' },
+            { order: '7', name: 'How we make users happy', to: '/handbook/making-users-happy' },
+            { order: '8', name: 'How we make money', to: '/handbook/how-we-make-money' },
+            { order: '9', name: "How we're building a world-class team", to: '/handbook/strong-team' },
+            { order: '10', name: 'Not running out of money', to: '/handbook/finance' },
+            { order: '11', name: 'How we got here', to: '/handbook/story' },
+            { order: '12', name: 'Where we are now', to: '/handbook/current-status' },
+            { order: '13', name: 'Where are we going?', to: '/handbook/future' },
+            { order: '14', name: 'How you can help', to: '/handbook/help' },
```

Comment:
I know we came up with this ordering, but it feels like jumping around here. Quick suggestion

```suggestion
            { order: '1', name: 'What is PostHog?', to: '/handbook/what-is-posthog' },
            { order: '2', name: 'How we got here', to: '/handbook/story' },
            { order: '3', name: 'How we get users', to: '/handbook/how-we-get-users' },
            { order: '4', name: 'How we make users happy', to: '/handbook/making-users-happy' },
            { order: '5', name: 'How we make money', to: '/handbook/how-we-make-money' },
            { order: '6', name: 'Enduringly low prices', to: '/handbook/low-prices' },
            { order: '7', name: 'Deciding which products to build', to: '/handbook/which-products' },
            { order: '8', name: 'A wide company with small teams', to: '/handbook/wide-company' },
            { order: '9', name: "How we're building a world-class team", to: '/handbook/strong-team' },
            {
                order: '10',
                name: 'Providing a world-class engineering environment',
                to: '/handbook/world-class-engineering',
            },
            { order: '11', name: 'Not running out of money', to: '/handbook/finance' },
            { order: '12', name: 'Where we are now', to: '/handbook/current-status' },
            { order: '13', name: 'Where are we going?', to: '/handbook/future' },
            { order: '14', name: 'How you can help', to: '/handbook/help' },
```